datasets/us/ofac/us_ofac_sdn.yml

```diff
@@ -441,3 +441,29 @@ lookups:
         value: female
       - match: MALE
         value: male
+  type.string: 
+    lowercase: true
+    normalize: true
+    options: 
+      - match: Codigo Postal 80000
```

Comment:
maybe just a string replace in the code could be nice here.

If you glance at this property, do you see more you can easily clean up without a lot of time on it?